### ISSUE TITLE
[Keyboard Navigation - Azure BOT Framework- Test in Web Chat]: Keyboard focus indicator is not visible on 'upload file' button present under Test in web chat screen.

### ISSUE THREAD

**Issue by KattaBhargav** [microsoft/BotFramework-WebChat#4302]
### User Experience:
If Keyboard focus indicator is not visible on the 'upload file' button then keyboard user are not able to locate the keyboard focus indicator hence navigation on the page gets difficult and keyboard user gets disoriented.

**Note:** 
User credentials should NOT be included in the bug.

### Repro Steps:
Step 1: Open https://ms.portal.azure.com/?microsoft_azure_marketplace_ItemHideKey=BotWithComposer&Microsoft_Azure_BotService=friendName#create/hub using v- in which access is provided by the product team.
Step 2: Navigate to search box and search "Azure Bot" and  select "Azure bot services".
Step 3: Navigate to Bot services window and select already created Bot.
Step 4: Navigate to "Test in Web Chat" list item under azure bot window and select it.
Step 5: Navigate all the controls of "Test in Web Chat" blade under Azure bot window.
Step 6: Verify the issue.

### Actual Result:
Keyboard focus indicator is not visible on 'upload file' button present under Test in web chat screen.

### Expected Result:
Keyboard focus indicator should be clearly visible on 'upload file' button present under Test in web chat screen. All four sides of an indicator should be clearly visible. 

**Note:** Same issue reproes for 'send' button under same screen.

### MAS Reference:
[MAS 2.4.7 – Focus Visible](https://aka.ms/MAS2.4.7)

### Reference Links:

- [Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-poyad%40microsoft.com%7cc11478591b5a474f6d9d08d7199726c0%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637006009300301864&sdata=ZTWYamT1yE7yko5QU35ZJLf/5ZydtDeGAKrQGUiugJs%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

- External Bug Process: If this bug belongs to the external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information, please use this [link](https://microsoft.sharepoint.com/teams/cea11y/SitePages/External-Bug-Process.aspx). “HCL Staff should not log any third-party external bugs. Should be routed to edad team”.

- Please reach out to the [C&AI Teams channel](https://nam06.safelinks.protection.outlook.com/?url=https://teams.microsoft.com/l/team/19%253a3cd071821b694f11b054d878453cd75f%2540thread.skype/conversations?groupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&data=02%7c01%7cv-dhti%40microsoft.com%7cf90880a914be401871f108d763233116%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637086876782361405&sdata=/ULS%2BbeqMjysCoiWHvFgAbSnuFJqxO3wtpHmlidPjbU%3D&reserved=0) for any process-related queries.

### Test Environment: 

- OS: Windows 11 Version 21H2 (OS build: 22581.200)
- Browser: Edge Version 102.0.1227.0 (Official build) dev (64-bit)
- URL: https://ms.portal.azure.com/?microsoft_azure_marketplace_ItemHideKey=BotWithComposer&Microsoft_Azure_BotService=friendName#create/hub
- MPAC Version: 9.375.0.1 (dev#066c8b2659.220415-1243) Signed
- Tool used: Accessibility insights for web

**Comment by KattaBhargav** [comment]
#A11yMAS;#A11ySev2;#HCL;#WCAG2.4.7;#Accessibility;#Win11-Edge(Chromium);#BotFrameworkWebChat-Aug21;#Linked:Bug75209;#LinkedBug;

**Comment by KattaBhargav** [comment]
Bug copied from Azurebot team Bug id:-[BUG 75209](https://fuselabs.visualstudio.com/BotFramework/_workitems/edit/75209)

**Comment by kattepalli** [comment]
Below comment from AzureBot team:-
**can't fix this from ABS side
we use [BotFramework-WebChat](https://github.com/microsoft/BotFramework-WebChat)
and the style is https://github.com/microsoft/BotFramework-WebChat/blob/1e8817e076dd7055e26e6efa554325cfe8702129/packages/component/src/Styles/StyleSet/SendBoxButton.ts#L33**

**Comment by axelsrz** [comment]
@compulim could you take a look at this?

**Comment by compulim** [comment]
This is fixed in Web Chat 4.15.1 with PR #4028. However, the "Test in Web Chat" blade in Azure Bot Services is using an older version of Web Chat 4.14.0, which does not have the fix.

![image](https://user-images.githubusercontent.com/1622400/172344703-18c6cc22-2f0e-45e5-9f9d-8c1d2b5286dd.png)

![image](https://user-images.githubusercontent.com/1622400/172344899-97097887-43c1-4fd7-a8b9-08996b5f42f6.png)

I will ping ABS team and see if when they can upgrade the version of Web Chat in Azure Portal.

**Comment by cwhitten** [comment]
Closing as fixed. Azure Bot Service needs to upgrade to the latest stable version of Web Chat.
